aggregator/src/aggregator/aggregation_job_continue.rs

```diff
@@ -215,53 +229,33 @@ impl VdafOps {
                 }
             })
             .with_last_continue_request_hash(request_hash);
-        tx.update_aggregation_job(&helper_aggregation_job).await?;
 
-        accumulator.flush_to_datastore(tx, vdaf).await?;
+        try_join!(
+            tx.update_aggregation_job(&helper_aggregation_job),
+            try_join_all(
+                report_aggregations
+                    .iter()
+                    .map(|ra| tx.update_report_aggregation(ra))
```

Comment:
I think it would be marginally more efficient to produce bulk-insert operations. For now, I'm happy pipelining a set of individual requests -- this is simpler and hasn't shown itself to be inefficient.

I recorded this idea in https://github.com/divviup/janus/issues/224#issuecomment-1511897659. I think we shouldn't make this change until we see evidence that the pipelined requests are the cause of a performance/resource issue.